packages/accounts/src/light-account/utils.ts

```diff
@@ -33,7 +33,7 @@ export const getDefaultLightAccountFactory = (chain: Chain): Address => {
     case arbitrumGoerli.id:
     case base.id:
     case baseGoerli.id:
-      return "0x000000893a26168158fbeadd9335be5bc96592e2";
+      return "0x000000893A26168158fbeaDD9335Be5bC96592E2";
```

Comment:
any reason we capitalized some letters different from before?

Reply:
just to make it look the same as the one on light account deployed addresses doc